src/lib/core/usecases.py

```diff
@@ -2207,3 +2207,23 @@ def execute(self):
             target_fps=self._target_fps,
         )
         self._response.data = extracted_paths
+
+
+class DownlaodAnnotationClassesUseCase(BaseUseCase):
+    def __init__(
+        self,
+        response: Response,
+        annotation_classes_repo: BaseManageableRepository,
+        destination: str,
```

Comment:
please change the name to something more specific like download_path or something else